tests/pytests_launcher.py

```diff
@@ -32,6 +32,9 @@
 
 # We must call this to ensure the code coverage file writes get flushed
 # https://github.com/nedbat/coveragepy/issues/310
-coverage._atexit()
+if (cov := coverage.Coverage.current()) is not None:
+    cov.stop()
+    cov.save()
```

Comment:
so this instead of coverate._atexit? do we need to probably update the comment? idk

Reply:
I updated the comment to be more clear 👍